website/tests/test_cve_cache.py

```
@@ -0,0 +1,448 @@
+        mock_fetch.assert_not_called()
+
+    def test_empty_string_input(self, clear_cache):
+        """Test that empty string input returns None without cache or API calls."""
+        with patch("website.cache.cve_cache.fetch_cve_score_from_api") as mock_fetch:
+            result = get_cached_cve_score("")
+
+        assert result is None
+        mock_fetch.assert_not_called()
+
+    @patch("website.cache.cve_cache.fetch_cve_score_from_api")
+    def test_timeout_propagates_correctly(self, mock_fetch, clear_cache, caplog):
```

Ruff (0.14.5)

Unused method argument: `clear_cache`

(ARG002)